### ISSUE TITLE
Connection Missing on Single Child (Wordtree)

### ISSUE THREAD

**Issue by jammin411** [google/google-visualization-issues#2519]
When creating a word tree and there is a single child, no connection is drawn.  The words appear right next to each other without a graphic to separate them.  If you hover over each word you do see that it is two separate words, however on the screen they are drawn so close together that they appear as one word rather than a parent word with a single child.  

An example can be found here.  Notice no connection between the words Eat and Kibble.  I simple removed Cats Eat Mice from the data to replicate this issue.  

https://jsfiddle.net/vmja851m/

**Comment by nbering** [comment]
This behaviour seems consistent with all the other multi-word segments, such as "cats in the cradle lyrics" or "cats are people too". It seems it only groups a piece together as a node in the tree if there is more than one, and it has more than one distinct child node.

**Comment by jammin411** [comment]
The issue I have though is that it does not clearly show them as being parent/child relations.  Instead, with no connector, it draws the two words next to one another and makes them appear as though they are one word.  

I am using this in a medical environment where a term can be a short phrase, never more than 2 words.  So something like the word "circulations" has a single child, in my context, called "goal".  What happens is that "circulations goal" looks to be one phrase unless you hover over one of the two words.  

In my jsfiddle example above, "eat kibble" looks like one word rather than a parent/child relation unless I hover over it.  

What I want to see is a connector drawn between the two so it is clear that they are parent/child rather than appearing as one word.  It should not matter how many children exist, the connector still needs to be drawn.

**Comment by nbering** [comment]
Have you tried the explicit format?

https://developers.google.com/chart/interactive/docs/gallery/wordtree#implicit-and-explicit-word-trees

**Comment by jammin411** [comment]
Yes, I've tried both methods and the same outcome.  Explicit is how I currently have it running on my end, which I prefer because I make the connections rather than the charting software doing it for me.  Below is a sample JSON dataset that I am using to drive these connections as well as the options I pass into the call.

[{"id":0,"childLabel":"diabetes","parent":-1,"size":5},{"id":2,"childLabel":"metformin","parent":0,"size":3},{"id":202,"childLabel":"glyburide","parent":2,"size":1},{"id":203,"childLabel":"1 day","parent":2,"size":1},{"id":204,"childLabel":"dm2","parent":2,"size":1},{"id":5,"childLabel":"glyburide","parent":0,"size":3},{"id":501,"childLabel":"metformin","parent":5,"size":1},{"id":6,"childLabel":"patient","parent":0,"size":3},{"id":601,"childLabel":"nka","parent":6,"size":1},{"id":602,"childLabel":"pain","parent":6,"size":1},{"id":604,"childLabel":"significant other","parent":6,"size":1},{"id":607,"childLabel":"doses","parent":6,"size":1},{"id":608,"childLabel":"plan","parent":6,"size":1},{"id":610,"childLabel":"verbal","parent":6,"size":1},{"id":7,"childLabel":"effects","parent":0,"size":3},{"id":701,"childLabel":"drug","parent":7,"size":1},{"id":702,"childLabel":"opioids","parent":7,"size":1},{"id":12,"childLabel":"evaluation","parent":0,"size":3},{"id":1201,"childLabel":"treatment","parent":12,"size":1},{"id":1203,"childLabel":"today","parent":12,"size":1},{"id":1204,"childLabel":"ed","parent":12,"size":1},{"id":1205,"childLabel":"bph","parent":12,"size":1},{"id":1207,"childLabel":"blood","parent":12,"size":1},{"id":1208,"childLabel":"chronic low back pain","parent":12,"size":1},{"id":1209,"childLabel":"cxr","parent":12,"size":1},{"id":13,"childLabel":"history","parent":0,"size":3},{"id":1303,"childLabel":"assessment","parent":13,"size":1},{"id":1304,"childLabel":"physical","parent":13,"size":1},{"id":1307,"childLabel":"falling","parent":13,"size":1},{"id":1308,"childLabel":"laryngeal cancer","parent":13,"size":1},{"id":1309,"childLabel":"laryngeal stenosis","parent":13,"size":1},{"id":1310,"childLabel":"patient","parent":13,"size":1}]

var options = {
                colors: ["#0275d8", "black", "black"],
                wordtree: {
                    format: "explicit",
                    type: "suffix",
                    word: "diabetes"
                }
